devops/charts/bg-monitoring/values.yaml

```diff
@@ -1,6 +1,67 @@
+timezone: Europe/Berlin
+#Monitoring server configuration
 app:
+  enabled: true #enable app deployment,secret,service,pvc
+  replicas: 1
+  image:
+    stream: true #enable annotation to look for stream image name
```

Comment:
Default should be false here

Reply:
since we dont have any image in any public registry the behavior should be dusscussed